src/atoms/TextArea/TextArea.example.mdx

```diff
@@ -0,0 +1,15 @@
+import TextArea from '@atoms/TextArea';
+
+# TextArea
+<br/>
+<p> A multi-line plain-text editing control component that you can insert a character count as props and it will not accept any input after that amount. </p>
+<br/>
+
+<p>600 Character Limit</p>
+<TextArea MaxChar={600}></TextArea>
```

Comment:
Usually for props we want the casing to be camel, so that we can differentiate them from the TagName easier.
i.e: `MaxChar` here would become `maxChar`.

For the purposes here, I think just renaming it to `max` is detailed enough.